lamp-small.yaml

```diff
@@ -44,6 +44,8 @@ parameters:
     constraints:
     - allowed_pattern: "^[a-zA-Z0-9.-_]{1,255}$"
       description: Must only contain alphanumeric characters, '-', '_', and '.'
+    - length: { min: 1, max: 16 }
+      description: User name must be between 6 and 8 characters
```

Comment:
Per the length constraint above, one will in fact be able to pass in a username of 4 characters and still violate the description.